src/extension.ts

```diff
@@ -38,9 +38,10 @@ export async function activateInternal(context: ExtensionContext, perfStats: { l
 	await callWithTelemetryAndErrorHandling('azure-account.activate', async (activateContext: IActionContext) => {
 		activateContext.telemetry.properties.isActivationEvent = 'true';
 		activateContext.telemetry.properties.activationTime = String((perfStats.loadEndTime - perfStats.loadStartTime) / 1000);
+		activateContext.telemetry.properties.authLibraryOnStartup = getSettingValue(authLibrarySetting) || 'undefined';
```

Comment:
I think you could just wrap this with a `String()`.  I don't have a strong preference, but just think it looks a bit cleaner.

Reply:
I didn't think of that, nifty!